incubator/kafka/requirements.yaml

```diff
@@ -1,4 +1,4 @@
 dependencies:
 - name: zookeeper
-  version: 0.1.1
+  version: 0.2.0
   repository: http://storage.googleapis.com/kubernetes-charts-incubator
```

Comment:
Please update to new URL: `https://kubernetes-charts-incubator.storage.googleapis.com`, update your cache (`helm repo add incubator https://kubernetes-charts-incubator.storage.googleapis.com`), then re-run `helm dep update incubator/kafka`